client/common/directives/ModelFieldWYSIWYG/ModelFieldWYSIWYG.js

```diff
@@ -22,21 +22,23 @@ angular.module('dashboard.directives.ModelFieldWYSIWYG', [
 })
 
 .directive('modelFieldWysiwygEdit', function($compile, $cookies, $timeout, $modal, Config, FileUploadService) {
-  function getTemplate() {
+  function getTemplate(scope) {
+    let fontSizeArr = [7, 10, 12, 14, 16, 18, 24];
+    let minSize = !scope.options.fontSizeRange.min > fontSizeArr.length && scope.options.fontSizeRange != undefined && !scope.options.fontSizeRange.min < 1 ? scope.options.fontSizeRange.min : 1;
+    let maxSize = !scope.options.fontSizeRange.max > fontSizeArr.length && scope.options.fontSizeRange != undefined && scope.options.fonSizeRange.min < scope.options.fontSizeRange.max ? scope.options.fontSizeRange.max : 7;
```

Comment:
@leolynn you need parentheses around your comparisons, otherwise the ! is negating the variable, not the boolean result. Alternatively use <=/>= instead of >/< with no negation.

You also have a typo in scope.options.fonSizeRange.min, and aren't checking if maxSize is less than 1.

Also, please break these into separate lines for better readability